checkov/secrets/plugins/entropy_keyword_combinator.py

```diff
@@ -200,7 +200,7 @@ def get_lines_from_same_object(
                 possible_keywords.add(raw_context.lines[j])
                 if self.is_object_start(raw_context=raw_context, idx=j):
                     return possible_keywords
-        return possible_keywords
+        return set()
```

Comment:
maybe we can just explain why is this part of the logic: 

```suggestion
        # No start of array detected, hence all found possible_keywords are irrelevant
        return set()
```